dashboard/src/client.ts

```diff
@@ -52,8 +51,10 @@ function connectPackageMonitor() {
   const socket = new WebSocket(url);
   socket.onmessage = (event: MessageEvent) => {
     const body = JSON.parse(event.data);
-    const data = api.MonitorResponseBodyFromJSON(body);
-    store.handleEvent(data);
+    const data = api.MonitorEventFromJSON(body);
+    if (data.event) {
+      store.handleEvent(data.event);
```

Comment:
Currently, I'm relying on the browser's default behavior: halting the event processing and logging the error when an issue arises. If we were to introduce a try-catch block, what additional actions should we take to handle the error? Since I had no plans beyond deferring to the browser's default behavior, I thought it wasn't necessary to include it.

Reply:
I'm going to merge, but if you come up with ideas let me know!

Reply:
I was just considering what we should do for the user experience. Error logs in the devtools isn't exactly ideal. But I guess this is good enough for now especially given how unlikely this is.

Reply:
Yeah that makes sense, maybe at some point we could build some kind of indicator or dialog raising connectivity issues. Let's keep that in mind.